frontend/packages/dev-console/integration-tests/tests/pipeline.scenario.ts

```diff
@@ -120,4 +137,59 @@ describe('Pipeline', async () => {
       true,
     );
   });
+
+  it('run pipeline and generate pipeline logs scenario', async () => {
+    let runName: string;
+    await switchPerspective(Perspective.Developer);
+    await browser.wait(until.visibilityOf(pageSidebar));
+    await pipelinecheckStatus();
+    await execSync(
+      `oc create -f ./packages/dev-console/integration-tests/views/sample-app-pipeline.yaml -n ${testName}`,
+    );
+    await browser.wait(until.visibilityOf(pipelineTableBody));
+    expect(pipelineTableBody.samplePipelineElement.isDisplayed()).toBe(true);
+
+    await pipelineFilter.click();
+    await pipelineFilter.sendKeys('sample-app-pipeline');
+    await browser.wait(until.visibilityOf(pipelineTableBody.samplePipelineElement));
+    await pipelineStartKebab.click();
+    await browser.wait(until.elementToBeClickable(pipelineStart));
+    browser.sleep(5000);
+    await pipelineStart.click();
+    await browser.wait(until.visibilityOf(pipelineRunOverview));
+    await pipelineOverviewName.getText().then(function(divText) {
+      runName = divText;
+    });
```

Comment:
You're just trying to capture this text, just use async/await to the intent it was created for...

```
    const runName = await pipelineOverviewName.getText().then(function(divText) {
      return divText;
    });
```